### ISSUE TITLE
[Bug]: Activating Action Menu expanding submenu on iOS and VoiceOver closes the trsy

### ISSUE THREAD

**Issue by nikkimk** [adobe/spectrum-web-components#5110]
### Code of conduct

- [x] I agree to follow this project's code of conduct.

### Impacted component(s)

`<sp-action-menu>`

### Expected behavior

Using the VoiceOver on iOS to test the [ActionMenu > Submenu story](https://opensource.adobe.com/spectrum-web-components/storybook/index.html?path=/story/action-menu--submenu), after opening the menu in a tray, double tapping "Select some items" the submenu should be expanded.

### Actual behavior

Using the VoiceOver on iOS to test the [ActionMenu > Submenu story](https://opensource.adobe.com/spectrum-web-components/storybook/index.html?path=/story/action-menu--submenu), after opening the menu in a tray, double tapping "Select some items" to activate the submenu closes the tray rather than opening the submenu. The submenu story is similarly broken without VoiceOver on.

### What browsers are you seeing the problem in?

Safari

### How can we reproduce this issue?

1. Using the VoiceOver on iOS to test the [ActionMenu > Submenu story](https://opensource.adobe.com/spectrum-web-components/storybook/index.html?path=/story/action-menu--submenu)
2. After opening the menu in a tray, double tap "Select some items" to open the submenu.
3. The tray closes rather than opening the submenu. 

The submenu story is similarly broken without VoiceOver on.

### Severity

SEV 1

**Comment by najikahalsema** [comment]
swc-636

**Comment by najikahalsema** [comment]
This is blocked because we don't have a specific design for mobile. We need to figure out what short term changes we can make in that context.
